{"name": "test_8a", "status": "failed", "statusDetails": {"message": "AssertionError: False is not true", "trace": "casatasks/test_task_specfit.py:94: in tearDown\n    self.assertTrue(len(self._tb.showcache()) == 0)\nE   AssertionError: False is not true"}, "description": " Test two gaussian + one polynomial image with estimates", "start": 1758043994389, "stop": 1758043994488, "uuid": "11b6c0e6-c92a-4a07-a11f-14d71a922e09", "historyId": "507da6ac52fa33fb4bb949453716c399", "testCaseId": "507da6ac52fa33fb4bb949453716c399", "fullName": "casatasks.test_task_specfit.specfit_test#test_8a", "labels": [{"name": "parentSuite", "value": "casatasks"}, {"name": "suite", "value": "test_task_specfit"}, {"name": "subSuite", "value": "specfit_test"}, {"name": "host", "value": "casa-rockylinux8-amd64-perf07"}, {"name": "thread", "value": "597558-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "casatasks.test_task_specfit"}], "titlePath": ["casatasks", "test_task_specfit.py", "specfit_test"]}